styles.css

```
@@ -0,0 +1,340 @@
+@import url("https://fonts.googleapis.com/css2?family=Anek+Telugu:wght@600&family=Inconsolata:wght@600&family=Roboto:ital,wght@1,100&display=swap");
+
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+@keyframes move {
+  from {
+    transform: translateX(-5px);
+  }
+  to {
+    transform: translateX(10px);
+  }
+}
+
+.fondo {
+  position: relative;
+  z-index: -1;
+}
+.fondo img {
+  position: absolute;
+  width: 100%;
+  height: 280px;
+}
+
+.navegation {
+  width: 100%;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  box-sizing: border-box;
+  padding: 0px 16px;
+  padding-right: 10px;
+  margin-bottom: 80px;
+  max-width: 1144px;
+}
+
+.menu {
+  display: none;
+}
+
+.hamburger {
+  width: 40px;
+  height: 26px;
+  background-color: white;
+  display: flex;
+  justify-content: center;
+  flex-direction: column;
+  align-items: center;
+  border: none;
+  border-radius: 20px;
+}
+
+.hamburger .bar {
+  width: 50%;
+  height: 4px;
+  background-color: #32e2f1;
+  margin: 1px;
+}
+.hamburger .bar-2 {
+  width: 60%;
+  height: 4px;
+  background-color: blue;
+  margin: 1px;
+}
+.hamburger .bar-3 {
+  width: 50%;
+  height: 4px;
+  background-color: #32e2f1;
+  margin: 1px;
+}
+
+.Sign {
+  display: none;
+}
+.body {
+  margin-left: 20px;
+}
+
+main {
+  width: 100%;
+}
+
+.two-columns {
+  width: 100%;
+  display: flex;
+  justify-content: center;
+}
+
+.hero {
+  width: 100%;
+  display: flex;
+  flex-direction: row;
+  justify-content: flex-start;
+  align-items: center;
+  box-sizing: border-box;
+}
+.two-columns p {
+  max-width: 480px;
+}
+
+.textos {
+  width: 100%;
+  height: auto;
+}
+.title {
+  font-family: "Anek Telugu", sans-serif;
+  font-weight: 600;
+  font-size: 50px;
+  margin: 0;
+  letter-spacing: -1px;
+  line-height: 60px;
+}
+
+p {
+  font-size: 18px;
+  line-height: 26px;
+  font-weight: 400;
+  font-family: "Arial", sans-serif;
+  margin:30px 0;
+  
+}
+
+.buttons {
+  display: flex;
+  margin: 16px 0;
+  align-items: center;
+}
+
+.start-button {
+  font-family: "Pangolin", cursive;
+  font-size: 16px;
+  letter-spacing: 1.25px;
+  font-weight: bold;
+  border-radius: 10px;
+  border: none;
+  background-color: darkorange;
+  color: white;
+  padding: 8px;
+}
+
+.start-button:hover {
+  background-color:darkcyan;
+}
+.start-button .icon:hover {
```

Nico, aquí le estás diciendo a CSS que aplique los estlos al hacer hover sobre solo el icono... y generalmente uno lo hace sobre la todo el botón, no solo sobre el icono porque es más pequeño... cambialo así:

```suggestion
.start-button:hover .icon {
```